src/langapi/Makefile

```diff
@@ -1,4 +1,4 @@
-SRC = mode.cpp language_ui.cpp languages.cpp language_util.cpp
```

Comment:
I removed it because it only contained a couple of methods, and it seemed cleaner to declare them inline. I'm not sure about the virtual table issue - what's the potential problem here?